src/System.Runtime/src/ApiCompatBaseline.uapaot.txt

```diff
@@ -244,5 +244,6 @@ CannotRemoveBaseTypeOrInterface : Type 'System.Threading.WaitHandle' does not in
 CannotRemoveBaseTypeOrInterface : Type 'System.Threading.Tasks.Task' does not implement interface 'System.IDisposable' in the implementation but it does in the contract.
 MembersMustExist : Member 'System.Threading.Tasks.Task.Dispose()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'System.Threading.Tasks.Task.Dispose(System.Boolean)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.Threading.LazyInitializer.EnsureInitialized<T>(T, System.Object, System.Func<T>)'
```

Comment:
I'm sorry. While cutting out the trailing csproj in compilation error, I chewed up a bit more. I should have seen other lines. Honestly, been doing silly mistakes over my limit. Need to introspect why.